src/json.hpp.re2c

```diff
@@ -6168,6 +6168,21 @@ class basic_json
               thousands_sep(!loc->thousands_sep ? '\0' : loc->thousands_sep[0]),
               decimal_point(!loc->decimal_point ? '\0' : loc->decimal_point[0])
         {}
+        serializer(const serializer& cpy)
```

Comment:
The serializer class is private and only used inside the `dump` function, so no code should rely on it. Doing nothing and relying on default behavior is fine (I could live with the `-Weffc++` warnings), but making explicit that we do not intent to copy serializers feels cleaner.